packages/web/src/app/Header.tsx

```diff
@@ -36,7 +36,7 @@ export default function Header() {
         {t('translate:interlinear')}
       </NavLink>
       {userCan('create', 'Language') && (
-        <NavLink to={'/admin'} className={navLinkClasses}>
+        <NavLink to={'/admin/languages'} className={navLinkClasses}>
```

Comment:
I could have implemented this dynamically somehow, so that it automatically goes to the first section of the admin page, no matter what it is. That seemed like unnecessary complexity.

Reply:
We actually need to do this the same way I did with the language settings view because the NavLink renders differently if the URL matches. That won't work for the users section of the admin view because it has a different path. Take a look at the languages/router.tsx file for how I've done this

Reply:
Ok I attempted something that seems to work with the loader, but I very well could have missed something.